packages/web/src/components/track/DynamicTrackArtwork.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  useGetTrackById,
+  SquareSizes,
+  statusIsNotFinalized,
+  ID
+} from '@audius/common'
+import cn from 'classnames'
+
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import { useTrackCoverArt2 } from 'hooks/useTrackCoverArt'
+
+import styles from './DynamicTrackArtwork.module.css'
+
+export enum DynamicTrackArtworkSize {
+  /** 40x40 */
+  SMALL = 'small',
+  /** 64x64 */
+  DEFAULT = 'default'
+}
+
+type DynamicTrackArtworkProps = {
+  id: ID
+  className?: string
+  size?: DynamicTrackArtworkSize
+}
+
+/** Loads artwork for a given track ID and applies an optional className */
+export const DynamicTrackArtwork = ({
```

Comment:
cc @dylanjeffers We had talked about having a component that can be used to just load artwork by track ID. This should take care of it and has two sizes based on the usage we have so far in feature work. We can formalize this with the Harmony TrackArtwork component.